src/ripple/app/ledger/InboundLedgers.h

```diff
@@ -63,7 +63,7 @@ class InboundLedgers
 
     virtual int getFetchCount (int& timeoutCount) = 0;
 
-    virtual void logFailure (uint256 const& h) = 0;
+    virtual void logFailure (uint256 const& h, std::uint32_t seq) = 0;
```

Comment:
If we want these type aliases to be used, they need bite. In other words, it should be a compile error if they are not used. That's what the "tagged integers" were for. Until we make `LedgerSeq` a tagged integer, this will keep happening.